nucleus/osgi-platforms/osgi-cli-interactive/pom.xml

```diff
@@ -62,13 +62,6 @@
         </developer>
     </developers>
 
-    <licenses>
-        <license>
-            <name>CDDL + GPL 1.1</name>
-            <url>http://glassfish.java.net/public/CDDL+GPL_1_1.html</url>
-        </license>
-    </licenses>
-
```

Comment:
Why is this license entry being removed?

Reply:
This is based on one of your review comments.
> #28 - Yes, the license information in the pom.xml file will need to be updated.  Or just remove it and inherit it from the parent.

Reply:
Ah, ok, that's what I suspected.  Thanks!